src/portable/instance/upgrade.rs

```diff
@@ -407,6 +407,15 @@ pub fn upgrade_incompatible(
                 paths.dump_path.join("edgedb.dump"),
                 paths.dump_path.join("main.dump"),
             )?;
+
+            if let Ok(mut creds) = credentials::read_sync(&paths.credentials) {
+                if creds.database == Some("edgedb".to_string()) {
+                    log::info!("Updating credentials file {:?}", &paths.credentials);
+                    creds.database = Some("main".to_string());
```

Comment:
Any drawback to unsetting the `database` property here?

Reply:
We currently have a bug where the CLI wants to rewrite the credentials if we omit it. Ideally we'd only have one or the other, yeah.